src/app.js

```diff
@@ -132,6 +133,18 @@ var app = function(options) {
     });
   }
 
+  if (options.docs == undefined) {
+    options.rootDocsRedirect = false;
+  }
+
+  if (options.rootDocsRedirect) {
+    let link = options.docs.documenter.getDocumentationUrl();
+    DOCS_HTML = '<html><body><a href='+link+'>Refer to the documentation</a></body></html>';
```

Comment:
We probably want to use a [template literal](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals) here. Do we want this page to be unstyled like this or to have a real template (I know the overuse of template is confusing here)?